docs/datamodel/access_policies.rst

```diff
@@ -205,28 +247,41 @@ Let's add a policy to our sample schema.
 
     +     access policy author_has_full_access
     +       allow all
-    +       using (global current_user ?= .author.id);
+    +       using (global current_user    ?= .author.id
+    +         and  global current_country ?= Country.Full) {
+    +        errmessage := "User not in country with full access";
+    +       }
+    +      access policy author_has_read_access
+    +        allow select
+    +        using (global current_user    ?= .author.id 
+    +          and  global current_country ?= Country.ReadOnly);
         }
 
-
-Let's break down the access policy syntax piece-by-piece. This policy grants
-full read-write access (``all``) to the ``author`` of each ``BlogPost``. No
-one else will be able to edit, delete, or view this post.
+Let's break down the access policy syntax piece-by-piece. These policies grant
+full read-write access (``all``) to the ``author`` of each ``BlogPost``, if
+the author is in a country that allows full access to the service. Otherwise,
+the same author will be restricted to either read-only access or no access at
+all, depending on the country.
 
 .. note::
 
   We're using the *coalescing equality* operator ``?=`` which returns
```

Comment:
Not your change, I know, but while we're here… What do you think?
```suggestion
  We're using the *coalescing equality* operator ``?=`` because it returns
```
This makes it sound less like a side effect and more like the intended effect (which I assume it was).